### DISCUSSION TITLE
Ansible- Error while installing Erlang & Rabbitmq

### DISCUSSION THREAD

**Discussion by Swarani2929** [learndevopsonline/learndevopsonline#739] [Q&A]
Hi @r-devops getting error while installing Erlang & Rabbitmq, kindly help. Below screenshot for reference.

<img width="947" alt="image" src="https://user-images.githubusercontent.com/109668941/235189393-9199f0de-99f5-4d56-aa3a-6a0e5a546efb.png">

<img width="960" alt="image" src="https://user-images.githubusercontent.com/109668941/235176733-eb8e16a9-f637-403a-b4fd-282e3504fa34.png">

**Comment by anandnandhu1** [comment]
is this issue solved ? and how

**Reply by Swarani2929** [reply]
I destroyed everything and rerun the infra, then it worked for me.
